src/plasoscaffolder/bll/services/sqlite_generator.py

```diff
@@ -21,33 +26,53 @@ def __init__(self, path, name, database, output):
     self.init_parser_exists = file_exists(parser_init_file_path(self.path))
     self.output = output
 
-  def generate_sqlite_plugin(self, fileHandler):
+  def generate_sqlite_plugin(self, fileHandler: FileHandler):
+    """
+    Generate the whole sqlite plugin
+
+    :param fileHandler: the Filehandler class
+    """
     file_handler = fileHandler()
 
     file = file_handler.create_file_from_path
     copy = file_handler.copy_file
     edit = file_handler.add_content
 
-    content_init_formatter = "from plaso.formatters import " + self.name
-    content_init_parser = "from plaso.parsers.sqlite_plugins import " + \
-                          self.name
+    content_init_formatter = get_formatter_init_edit(
+      self.name) if self.init_formatter_exists \
```

Comment:
use 4 space indentation after ( line break

I opt not to use line continuation with \ for readability.
The short hand is harder to debug I cannot easily tell which function raises.
```
if self.init_formatter_exists:
  content_init_formatter = get_formatter_init_edit(self.name)
else:
  content_init_formatter = get_formatter_init_create(self.name)
```